js/ui/calendar/ui.calendar.js

```diff
@@ -732,14 +780,21 @@ const Calendar = Editor.inherit({
     },
 
     _translateViews: function() {
-        move(this._view.$element(), { left: 0, top: 0 });
+        const views = this.option('views');
+
+        move(this._view.$element(), { left: 0, top: 0 }); // 0
 
         this._beforeView && move(this._beforeView.$element(), {
             left: this._getViewPosition(-1),
             top: 0
         });
 
         this._afterView && move(this._afterView.$element(), {
+            left: this._getViewPosition(views),
+            top: 0
+        });
+
+        this._additionalView && move(this._additionalView.$element(), {
```

Comment:
We may extract this construction in separate function and use it for all views
```
_moveViewElement(view, coefficient) {
        view && move(view.$element(), {
            left: this._getViewPosition(coefficient),
            top: 0
        });
}
```

Reply:
seems good